master/buildbot/test/fake/fakedb.py

```diff
@@ -528,6 +533,7 @@ class LogChunk(Row):
         compressed=0)
 
     required_columns = ('logid', )
+    utf8_columns = ('content', )
```

Comment:
Yes, comment is a good idea.
Do you think it's better to place it here, or in `Raw` class near `Raw.utf8_columns` definition?